apps/api/src/app.module.ts

```diff
@@ -61,6 +69,12 @@ const modules: Array<Type | DynamicModule | Promise<DynamicModule> | ForwardRefe
   TenantModule,
 ];
 
+const eeAuthModule = importWhen({ path: '@novu/ee-auth', condition: process.env.NOVU_MANAGED_SERVICE === 'true' });
```

Comment:
Would suggest maybe considering even a higher level function to encapsulat this logic, for example `enterpriseImport('@novu/ee-auth')`